Packs/BitwardenPasswordManager/Integrations/BitwardenPasswordManager/BitwardenPasswordManager.py

```diff
@@ -46,9 +46,11 @@ def is_token_expired(self, expires_date: str) -> bool:
         return utc_now > expires_datetime
 
     def create_new_token(self, json_data: dict) -> str:
+        full_url = AUTHENTICATION_FULL_URL.replace(".com", ".eu") if ".eu" in self._base_url else AUTHENTICATION_FULL_URL
```

Comment:
There is a chance that `.eu` will show in the base url not as the Europe suffix, for example a company that their url is `www.eusomething.com`. Do we want to handle this?
According to the documentation, are there other auth URL for more regions? OR just `.com` and `.eu`?